upstream-community-operators/infinispan/infinispan.package.yaml

```
@@ -1,4 +1,6 @@
 packageName: infinispan
 channels:
```

```suggestion
defaultChannel: stable
channels:
```

Whenever there is more than one channel, a `defaultChannel` needs to be specified.